main.py

```
@@ -1,572 +1,100 @@
-        end_time = original_video_length
-
-    if start_time < 0 or start_time > original_video_length:
-        raise ValueError(f"start_time must be between 0 and {original_video_length}")
-    if end_time < 0 or end_time > original_video_length:
-        raise ValueError(f"end_time must be between 0 and {original_video_length}")
-    if start_time >= end_time:
-        raise ValueError(f"start_time must be less than end_time")
+def process_video_url(
+    video_url: str, 
+    start_time: float = 0, 
+    end_time: float = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)